Addresses publish error by removing runpath for CC.
Will not send upstream.

--- ltmain.sh	2016-11-14 13:28:02.161103599 +0000
+++ ltmain.sh	2016-11-14 14:11:06.669510804 +0000
@@ -7332,7 +7332,8 @@
 	save_ifs="$IFS"; IFS='~'
 	for cmd in $cmds; do
 	  IFS="$save_ifs"
-	  eval cmd=\"$cmd\"
+	    # Append the command to remove runpath for CC.
+	  eval cmd=\"$cmd -norunpath \"
 	  $opt_silent || {
 	    func_quote_for_expand "$cmd"
 	    eval "func_echo $func_quote_for_expand_result"
